src/app/models/reference.ts

```diff
@@ -610,3 +610,19 @@ export function getRefListAuthorsNames(ref: Reference): string {
 export function getRefNodeText(ref: Reference): string {
   return [getRefListAuthorsNames(ref), get(ref.referenceInfo, 'year')].filter(Boolean).join(', ');
 }
+
+export function sortReferencesList(refs: Reference[]): void {
+  refs.sort((ref1, ref2) => {
+    return getAuthorLastNameForSorting(ref1) < getAuthorLastNameForSorting(ref2)
```

Comment:
I personally think stacking ternary operators as such makes the code a little hard to read.